webapp/src/main/resources/db/migration/V66__Remove_workdir_prefix_from_usages_column.sql

```diff
@@ -0,0 +1 @@
+UPDATE asset_text_unit_usages atuu INNER JOIN asset_text_unit atu ON atu.id = atuu.asset_text_unit_id INNER JOIN branch b ON b.id = atu.branch_id INNER JOIN repository r ON r.id = b.repository_id SET usages = SUBSTRING(atuu.usages, LENGTH("/workdir/") + 1) WHERE atuu.usages LIKE "/workdir/%" AND r.name = "webapp";
```

Comment:
This shouldn't be checked in as part of the OSS solution, as this is strictly to do with a data quality issue on our specific instance/installation.